sonic_y_cable/y_cable.py

```diff
@@ -1350,8 +1356,45 @@ def upgrade_firmware(physical_port, fwfile):
              a string, a path to the binary file which contains the firmware image
     Returns:
         an Integer:
-             a predefined code stating whether the firmware upgrade was successful
-             or an error code as to what was the cause of firmware upgrade failure
+             a predefined code stating whether the firmware download was successful
+             or an error code as to what was the cause of firmware download failure
     """
 
     return FIRMWARE_UPGRADE_SUCCESS
+
+def activate_firmware(physical_port):
+    """ This routine should basically activate the downloaded firmware
+    of the Y cable on all the componenets of the Y cable of the port specified.
```

Comment:
Remove the first 'of the Y cable'; redundant.

Reply:
fixed